projects/sample-native-extension/src/lib/employee-only-store-guard/README.md

```diff
@@ -1,108 +0,0 @@
-# Employee Only Store Guard
```

Comment:
Did you mean to remove these components?

I think it is fair to remove the KlarnaCheckoutComponent but the Employee Store Guard and Inject script service would still be useful

Reply:
good catch, I was a bit too rough on my Delete here. will put them back...

Reply:
@adriaan-venter all done.